### PR TITLE
Gallery: Reduce computation time by a factor 10

### PR DESCRIPTION
**Pull request by sciunto**
## Description

Quick enhancement. I reduced by a factor 10 the computation time of an example in the gallery. Just cropped the image asap.

### CODE REVIEW

File: doc/examples/segmentation/plot_extrema.py [hunk 1]
```diff
@@ -19,19 +19,20 @@
...
 color_image = data.hubble_deep_field()
-img = color.rgb2grey(color_image)
 
-# the rescaling is done only for visualization purpose.
-# the algorithms would work identically in an unscaled version of the
-# image. However, the parameter h needs to be adapted to the scale.
-img = exposure.rescale_intensity(img)
-
-# for visualization, we work on a crop of the image.
+# for illutration purposes, we work on a crop of the image.
```
Comment (soupault) [inline doc/examples/segmentation/plot_extrema.py]: `illu_s_tration`

Comment (sciunto) [inline doc/examples/segmentation/plot_extrema.py]: typo fixed!

**Comment by NelleV** [comment]
Reading the title of the PR, I thought it was a gain on the whole gallery building… I was excited to discover the trick :(

**Comment by soupault** [comment]
@sciunto Great markering skills indeed! :smile: You should definitely consider being a release manager for the next version of the library!

**Comment by jni** [comment]
This is a great example of what I meant by maintainers trying to change titles of merged PRs so that they make sense a year later... =D I've named the merge commit "Reduce runtime of local_maxima gallery example". ;)

**Comment by sciunto** [comment]
Proud to serve as an example :)

**Pull request merged**